### ISSUE TITLE
Add alert message after success or failure of a file deletion

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#78072]
## Issue Description
Originated from [Accessibility Feedback ticket: Delete button removes upload without user confirmation](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/59263), which gave two recommendations:

1. Some kind of status message that receives focus when an issue has been removed, similar to the background-only alerts in use when changing contact information. If a visible status message isn't practical, at least some aria-live update should be announced to screen reader users.
2. Some kind of intermediate confirmation step between a user clicking Delete and the issue actually being removed. I believe the team working on the Supplemental Claims Form 20-0995 is exploring using a modal and intends to contribute back to the pattern in the design system.
  
Design in [Design for deleting an uploaded file](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/63414).

We verified in the design ticket that the confirmation dialog already exists in staging. The design ticket adds an alert "feedback message".

This [delete confirmation](https://www.figma.com/file/rG5tNh3dS8FUoc5jtDshUk/526EZ-BDD-SHA-updates?type=design&node-id=15-3412&mode=design&t=XEeRWppIU7SFnyWo-11) should be shown after clicking "yes, delete this file" from the delete confirmation dialog in both the regular 526 and BDD flows. Design is based on the [content style guide page on feedback messages](https://design.va.gov/content-style-guide/error-messages/feedback), which is specific about the language and alert placement for a delete confirmation.

See the note in Figma of:

> After clicking “yes, delete this file” in the delete confirmation dialog, scroll to the top of the page and focus on the alert to make the confirmation or error message apparent both visually and for screen readers.
---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
